tests/Unit/Listeners/SendVoucherHistoryEmailTest.php

```diff
@@ -91,7 +91,7 @@ public function testRequestVoucherHistoryEmail()
         $this->seeEmailWasSent()
             ->seeEmailTo($user->email)
             ->seeEmailSubject('Rose Voucher Payment Records')
-            ->seeEmailContains('Hi ' . $user->name)
+            ->seeEmailContains('Hi ' . e($user->name))
```

Comment:
What does e do/ stand for and which package is the function from?

Reply:
It's a laravel helper function https://laravel.com/docs/5.4/helpers#method-e

Reply:
A nice - is it worth a comment or explicit use of htmlspecialchars? Could be hard to track down in future

Reply:
I'll add a comment in = )

Reply:
Sure - no big deal - was curious. Won't hurt. :)